### DISCUSSION TITLE
Dark Mode Won't Turn Off

### DISCUSSION THREAD

**Discussion by watrall** [requarks/wiki#4048] [Error / Bug Report] [answered]
Just upgraded my version on DigitalOcean (to 2.5.201).  When it restarted, it was in dark mode.  Went to the Themes section, and the dark mode toggle was off.  Turned it on (to change because the site was already in dark mode) and saved it.  Then, turned dark mode off (which changed the site back to light mode) and saved.  Went back to the landing page and the site turned back to dark mode.  Went back to themes, and dark mode was still turned off.  

Update - just discovered that if I open the admin area, go to themes, and then go to any of the other admin area pages, light mode turns back on - and stays on as long as I'm in the admin area. As soon as I exit the admin area, dark mode gets turned back on.

**Comment by luanaraujo** [comment]
I'm having the same issue.

**Comment by fnafox** [comment]
The option in admin area is just the default setting which can be overriden per user.
Check in your user profile if dark mode is set there.

**Comment by luanaraujo** [comment]
there is no theme option in my user profile page.

**Comment by luanaraujo** [comment]
Another question, is normal to have only one option here?

![image](https://github.com/requarks/wiki/assets/50503859/00d218b4-3800-47ed-9e0a-eb4a86bbfaa2)

Mine only shows Default

**Comment by NGPixel** [comment, marked as answer]
Dark mode is set in 2 locations:

- Admin > Theme > Dark Mode
- Profile >Appearance

Make sure the first one is disabled and the second one is set to either Site Default or Light mode.
